imap_processing/hit/l0/decom_hit.py

```diff
@@ -197,6 +101,13 @@ def parse_count_rates(sci_dataset: xr.Dataset) -> None:
             dims = ["epoch"]
```

Comment:
Do you think you'll need to add more to this list? If not and it's only being u sed in that dims loop, it might be better to directly set the epoch coordinate